src/basic/capability-util.c

```diff
@@ -271,12 +274,15 @@ int capability_bounding_set_drop_usermode(uint64_t keep) {
 
         r = drop_from_file("/proc/sys/kernel/usermodehelper/inheritable", keep);
         if (r < 0)
-                return r;
+                goto finish;
 
         r = drop_from_file("/proc/sys/kernel/usermodehelper/bset", keep);
         if (r < 0)
-                return r;
+                goto finish;
 
+finish:
+        if (r < 0 && errno == EPERM && (detect_container() > 0))
+                return 0;
         return r;
 }
```

Comment:
I think it would be better to guard this one function on the caller side, and bind it to the availability of the right kind of cap?